update-issues/main.js

```diff
@@ -193,12 +208,12 @@ async function maintainMilestones(issue, nextMilestone, backlog) {
 
 const ALL_ISSUE_TASKS = [
   reopenIssues,
-  archiveOldIssues,
-  unarchiveIssues,
-  manageWaitingIssues,
-  cleanUpIssueTags,
+  archiveOldIssuesAndPRs,
+  unarchiveIssuesAndPRs,
+  manageWaitingIssuesAndPRs,
+  cleanUpIssueAndPRTags,
   pingQuestions,
-  maintainMilestones,
+  maintainIssueMilestones,
 ];
 
 async function processIssues(issues, nextMilestone, backlog) {
```

Comment:
Okay, it's a small note, but shouldn't this method now be called `processIssuesAndPRs`? Renaming every single thing that could deal with an issue or a PR to "issueAndPr" or whatever would be overkill, but since this method directly calls a bunch of "IssuesAndPRs" methods it might be worth renaming.

Also the logs could say "issue #whatever" for issues and "pull request #whatever" for PRs.

Reply:
I renamed processIssues and ALL_ISSUE_TASKS, and I added comments.  I also updated the formatting to account for isPR.